shakenfist/virt.py

```diff
@@ -285,12 +296,43 @@ def video(self):
     def version(self):
         return self.__version
 
+    # Values routed to attributes, writes are via helper methods.
```

Comment:
I feel attribute setters should be defined if needed, rather than using helper methods elsewhere. The code would be safer if get/set are strictly controlled.

Reply:
The helpers don't fit the signature of a setter is why they're done this way at the moment. We can work through that though? I think we could pass a tuple to a setter for example.